### PR TITLE
Updates to refactored apps

### PR DESCRIPTION
**Pull request by ericharmeling**
This PR includes the following changes to the refactored sample apps:

- Removed subprocesses from refactored sample apps
- Changed local cluster from cockroach demo to cockroach start-single-node (see https://github.com/cockroachdb/docs/issues/7816#issuecomment-908552149 for explanation)
- Added includes for cluster startup and database initialization
- Updated cockroach sql file commands to be windows-compatible

All of these changes should reflect merged updates to the corresponding sample app repos. Note that I opened a separate PR for the recently-refactored Django sample app (https://github.com/cockroachdb/docs/pull/11620).

After this PR is approved, I will propagate the changes to the 20.2 and 21.1 docsets.

### CODE REVIEW

File: _includes/v21.2/app/init-bank-sample.md [hunk 1]
```diff
@@ -1,25 +1,26 @@
 
 To initialize the example database, use the [`cockroach sql`](cockroach-sql.html) command to execute the SQL statements in the `dbinit.sql` file:
 
+<div class="filter-content" markdown="1" data-scope="cockroachcloud">
+
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cat dbinit.sql | cockroach sql --url "<connection-string>"
+cat dbinit.sql | cockroach sql --url '<connection-string>'
```
Comment (ianjevans) [inline _includes/v21.2/app/init-bank-sample.md:8]: This connection string requires double quotes, I'm pretty sure, as we include environment variables in the connection string now.

File: _includes/v21.2/app/init-bank-sample.md [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
 ~~~
 
-Where `<connection-string>` is the connection string to the running cluster.
-
-<div class="filter-content" markdown="1" data-scope="cockroachcloud">
-
-{{site.data.alerts.callout_success}}
-Use the connection string you obtained earlier from the CockroachCloud Console.
-{{site.data.alerts.end}}
+Where `<connection-string>` is the connection string you obtained earlier from the CockroachCloud Console.
```
Comment (ianjevans) [inline _includes/v21.2/app/init-bank-sample.md:11]: See above re: single vs. double quotes.

File: v21.2/build-a-java-app-with-cockroachdb.md [hunk 3]
```diff
@@ -157,27 +113,28 @@ It does all of the above using the practices we recommend for using JDBC with Co
...
 
-To initialize the example database, pass the `dbinit.sql` file to the [`cockroach sql`](cockroach-sql.html) command:
+To initialize the example database, use the [`cockroach sql`](cockroach-sql.html) command to execute the SQL statements in the `dbinit.sql` file:
+
+<div class="filter-content" markdown="1" data-scope="cockroachcloud">
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cockroach sql --url '<connection-string>' -f app/src/main/resources/dbinit.sql
+cat app/src/main/resources/dbinit.sql | cockroach sql --url '<connection-string>'
```
Comment (ianjevans) [inline v21.2/build-a-java-app-with-cockroachdb.md:122]: Same issue re: single vs. double quotes.

File: v21.2/build-a-java-app-with-cockroachdb.md [hunk 4]
```diff
@@ -157,27 +113,28 @@ It does all of the above using the practices we recommend for using JDBC with Co
...
 ~~~
 
-Where `<connection-string>` is the connection string to the running cluster.
-
-<div class="filter-content" markdown="1" data-scope="cockroachcloud">
-
-{{site.data.alerts.callout_success}}
-Use the connection string you obtained earlier from the CockroachCloud Console.
-{{site.data.alerts.end}}
+Where `<connection-string>` is the connection string you obtained earlier from the CockroachCloud Console.
```
Comment (ianjevans) [inline v21.2/build-a-java-app-with-cockroachdb.md:125]: Same issue re: single vs. double quotes.

**Review by ianjevans — CHANGES_REQUESTED** [review]
Just some issues with the Cloud connection strings needing double quotes. Otherwise looks good.

File: v21.2/build-a-java-app-with-cockroachdb.md (hunk 3 above)
Comment (ericharmeling) [reply v21.2/build-a-java-app-with-cockroachdb.md]: Done.

File: _includes/v21.2/app/init-bank-sample.md (hunk 1 above)
Comment (ericharmeling) [reply _includes/v21.2/app/init-bank-sample.md]: Good catch. Done.

**Review by ianjevans — APPROVED** [review]

**Comment by ericharmeling** [comment]
TFTR !

**Pull request merged**